test/fixtures/get-test-server.ts

```diff
@@ -1,4 +1,5 @@
 import { createFake as createFakeDevicedb } from "@seamapi/fake-devicedb"
+import { paramsSerializer } from "@seamapi/http/connect"
```

Comment:
@seveibar This may deserve its own package at some point. Or make this module its own package under `@seamapi/axios`  https://github.com/seamapi/javascript-http/blob/main/src/lib/seam/connect/client.ts